File: VebProdavnica/VebProdavnicaDecijeIgracke/VPDecijeIgracke/VPDecijeIgracke/Models/StavkaPorudzbineModel/StavkaPorudzbine.cs
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using VPDecijeIgracke.Models.PorudzbinaModel;
using VPDecijeIgracke.Models.ProizvodModel;

namespace VPDecijeIgracke.Models.StavkaPorudzbineModel
{
    public class StavkaPorudzbine
    {

        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int StavkaID { get; set; }
        public decimal CenaStavka { get; set; }

        public int KolicinaStavka { get; set; }

        //strani kljucevi
        public int ProizvodID { get; set; }
        [ForeignKey("ProizvodID")]
        public Proizvod Proizvod { get; set; }

        public int PorudzbinaID { get; set; }
        [ForeignKey("PorudzbinaID")]
        public Porudzbina Porudzbina { get; set; }
    }
}

File: FrontendVebProdavnica/src/app/shared/models/stavkaPorudzbine.ts
export class StavkaPorudzbineCreationDTO {
  cenaStavka: number;
  kolicinaStavka: number;
  proizvodID: number;
  porudzbinaID: number;

  constructor(
    cenaStavka: number,
    kolicinaStavka: number,
    proizvodID: number,
    porudzbinaID: number
  ) {
    this.cenaStavka = cenaStavka;
    this.kolicinaStavka = kolicinaStavka;
    this.proizvodID = proizvodID;
    this.porudzbinaID = porudzbinaID;
  }
}
